cmd/kubeadm/app/componentconfigs/kubeproxy_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	clientsetfake "k8s.io/client-go/kubernetes/fake"
-	componentbaseconfig "k8s.io/component-base/config/v1alpha1"
+	componentbaseconfig "k8s.io/component-base/config/v1beta1"
```

Comment:
note to self / not requesting a change for this PR:
i guess this kubeadm test file can drop the import if it compares marshaled data.